packages/oui-field/README.md

```diff
@@ -0,0 +1,119 @@
+# oui-field
+
+<component-status cx-design="partial" ux="beta"></component-status>
+
+oui-field is a package which provides styles for form fields.
+
+## Installation
+
+```less
+  @import 'oui-form/form';
+```
+
+## Usage
+
+### Standard sizes
+
+```html:preview
+<div class="oui-field">
+  <label for="text" class="oui-field__label oui-label">Input size 1</label>
+  <div class="oui-field__content">
+    <div class="oui-field-control oui-field-control_1">
+      <input type="text" id="text" name="text" class="oui-input" value="Input text" />
+    </div>
+    <div class="oui-field-helper">An input with oui-field-control_1 class</div>
+  </div>
+</div>
+
+<div class="oui-field">
+  <label for="text" class="oui-field__label oui-label">Input size 2</label>
+  <div class="oui-field__content">
+    <div class="oui-field-control oui-field-control_2">
+      <label class="oui-select">
+        <select id="select" name="select" class="oui-select__input">
+          <option value="Option 1">Option 1</option>
+          <option value="Option 2">Option 2</option>
+          <option value="Option 3">Option 3</option>
+        </select>
+        <i class="oui-icon oui-icon-chevron-down" aria-hidden="true"></i>
+      </label>
+    </div>
+    <div class="oui-field-helper">
```

Comment:
I suppose the `oui-field-helper` is optional ? Can we add an example of it please ?